frontend/src/scenes/notebooks/Nodes/notebookNodeLogic.ts

```diff
@@ -70,6 +71,11 @@ export const notebookNodeLogic = kea<notebookNodeLogicType>([
             logic.values.editor?.insertContentAfterNode(props.getPos(), content)
         },
 
+        deleteNode: () => {
+            const logic = values.notebookLogic
+            logic.values.editor?.deleteRange({ from: props.getPos(), to: props.getPos() + 1 }).run()
```

Comment:
Not sure it will work in this situation because the node may not be selected but would it be possible to use the TipTap `deleteNode` command: https://tiptap.dev/api/commands/delete-node

Will the length of a custom node always be 1 or should we be doing `to: props.getPos() + node.nodeSize` instead?

Reply:
These are all good questions. The deleteNode one didn't seem to work as it would have to also be selected. From what I could see a block node like this will always have 1 "length" but yeah swapping to using a proper value is safer.
<img width="318" alt="Screenshot 2023-08-04 at 14 52 41" src="https://github.com/PostHog/posthog/assets/2536520/5f813834-fd88-49b1-93d3-c1d9e98ab279">

Reply:
We can probably get away without it for now, given `nodeSize` would be very easy to add in future and there hasn't been a real need for it anywhere else.

the `deleteRange` command always seemed a little abstract to me. It's used in a couple of places and given it's part of the editor API we expect people to use I might look into whether there is a better command we can expose